recipes/libsndfile/all/test_package/test_package.cpp

```diff
@@ -1,5 +1,9 @@
 #include "sndfile.hh"
 
+#if __cplusplus < 201100
+#undef nullptr
```

Comment:
I don't like public headers redefining built-in types https://github.com/libsndfile/libsndfile/blob/v1.0.30/include/sndfile.hh#L62